### PR TITLE
step-1 update intro

### PR DESCRIPTION
**Pull request by kostyfisik**
Initial solution of #3218

### CODE REVIEW

**Comment by davydden** [comment]
oh, i see what you mean now. Looks good to me, thanks :+1:

**Comment by tjhei** [comment]
A better place would probably be the top of http://www.math.tamu.edu/~bangerth/videos.html or at http://www.math.tamu.edu/~bangerth/videos.676.1.html @bangerth what do you say?

File: examples/step-1/doc/intro.dox [hunk 1]
```diff
@@ -128,6 +128,11 @@ command line, you may be interested in watching lectures 2.9 and 2.91.
 (@dealiiVideoLectureSeeAlso{2.9,2.91}) These give overviews over the command
 line and on what happens when compiling programs, respectively.
 
+Note that deal.II is actively developed, some functions were renamed
```
Comment (bangerth) [inline examples/step-1/doc/intro.dox]: can you write this sentence as "...developed, and in the course of this development we occasionally rename or deprecate functions or classes that are, however, still referenced in these video lectures."?

**Comment by bangerth** [comment]
I think we can point this kind of problem out in both venues.

File: examples/step-1/doc/intro.dox [hunk 2]
```diff
@@ -128,6 +128,13 @@ command line, you may be interested in watching lectures 2.9 and 2.91.
 (@dealiiVideoLectureSeeAlso{2.9,2.91}) These give overviews over the command
 line and on what happens when compiling programs, respectively.
 
+Note that deal.II is actively developed, and in the course of this
+development we occasionally rename or deprecate functions or classes
+that are still referenced in these video lectures.  For
+example, step-1 code shown in the video lecture 5 has a class
```
Comment (bangerth) [inline examples/step-1/doc/intro.dox]: -> the step-1 code shown in video lecture 5 uses a class

Comment (kostyfisik) [inline examples/step-1/doc/intro.dox]: Sorry, I am not a native English speaker :)

Comment (bangerth) [inline examples/step-1/doc/intro.dox]: I know -- neither am I, but at least my native language uses "the" and "a" in a broadly similar way to English :-)

**Pull request merged**